QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002137	$I've completed the first stage magic stone. {nl}It still does not satisfy the previous objective of delivering the perfect magic and magical power somewhat, {nl}but now at last it is possible to accept Link Magic even without magical ability.{nl}
+QUEST_20150714_002138	$Plan
+QUEST_20150714_002139	$If a magician finds a way to share all his abilities and knowledge to another, even a person without magical abilities can become a strong magician within a short period of time.
+QUEST_20150714_002140	$This way, you'd be able to hide from the eyes of Helgasercle who is extremely afraid of the disciple of Agailla Flurry appearing.
+QUEST_20150714_002141	$However, I'm not sure if this is at all possible or won't have any side effects.
+QUEST_20150714_002142	$Design Plan
+QUEST_20150714_002143	$The magic stone of the interaction can be worked on behalf of Link. {nl}However, maintaining the stone of interaction will require a considerable amount of magical power.
+QUEST_20150714_002144	$Hence, we decided to maintain the magic stone of the interaction with 12 regenerating magical power stones. {nl}If no problems arise, we will be able to maintain it, but if the magical power supply is cut, it won't stand for a long time.
+QUEST_20150714_002145	$Notice Board of Adrius
+QUEST_20150714_002146	$I am looking for a stone rubbing of the tombstone at the central sanctum. {nl}If you find any tombstones with writings engraved, make a rubbing off of it. {nl}- Adrius
+QUEST_20150714_002147	$The Tombstone with a Stain
+QUEST_20150714_002148	$It seems that someone has already made a rubbing since there is a vague trace of ink. {nl}Shall we read it?
+QUEST_20150714_002149	$King Kadumel finally dies.
+QUEST_20150714_002150	$Instead of Mage Tower, the Tower of Fire was a more fitting name for Flurry's tower.
+QUEST_20150714_002151	$This is a very secluded area so monsters usually prey around here. {nl}Be careful around here.
+QUEST_20150714_002152	$The Research Records of Demetrius
+QUEST_20150714_002153	$In ancient times, humans lived with the Goddesses. {nl}When I first heard of this folklore, I thought it was made up in the respect for the Goddesses. {nl}I started to investigate what historic background caused such reverence as well as why this myth was created.
+QUEST_20150714_002154	$I increased the extent of my research. {nl}I travelled to many regions of the kingdom and collected similar cases. {nl}And I started to believe that this story was part of a bigger truth rather than just a small truth covered in fabrication.
+QUEST_20150714_002155	$The facts that we've discovered so far are thus: {nl}{nl}In the beginning, or a period not that far from the beginning, all humans used to live with the Goddesses underground. {nl}I suppose the Goddesses ruled over humans with more direct care. {nl}I have no idea why but soon, the humans left that place and scattered across the land.
```

Comment:
"I suppose the Goddesses were more directly involved with humans. {nl}I have no idea why, but soon the humans left that place and scattered across the land."